common/src/main/java/com/wynntils/models/items/properties/RequirementItemProperty.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.properties;
+
+public interface RequirementItemProperty {
+    /**
+     * @return true if the item's lore has no X's in it, for the requirements section
```

Comment:
```suggestion
     * @return true if the item is usable by the player
```

Reply:
Although your comment is true, it is an implementation detail, and probably not the info that we should hold here.